infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/infrastructure/CorfuReplicationDiscoveryService.java

```diff
@@ -188,11 +188,13 @@ public void run() {
                 }
             }
         } catch (Exception e) {
-            log.error("Unhandled exception caught during log replication service discovery. Retry,", e);
+            log.error("Unhandled exception caught during log replication service discovery.", e);
```

Comment:
is it a good idea to add a todo for revisiting these exceptions handling which we discussed today?

Reply:
definitely, so we don't forget it. Let me add it.